### PR TITLE
Update BIOS to public v4.0 link

### PR DESCRIPTION
**Pull request by stripes416**
Signed-off-by: Kelly Ledford <kelly.ledford@intel.com>

### CODE REVIEW

File: getting_started/setup/setup_up_2_board.rst [hunk 1]
```diff
@@ -16,20 +16,15 @@ version is not currently supported.
...
-BIOS v3.6 added the audio OEM key for SOF. The BIOS main menu
-will show UP-APL01 R3.6.
+The BIOS main menu will show UP-APL01 R4.0.
 
-* Download the `BIOS <https://git-amr-4.devtools.intel.com/gerrit/gitweb?p=otc_audio-board-integration.git;tflink=projects.otc_audio/scm.Board_Integration>`__.
-
-  .. todo::
-   
-     this link needs to be updated to something accessible externally
+* Download the `BIOS <https://downloads.up-community.org/download/up-squared-uefi-bios-v4-0/>`__.
```
Comment (stripes416) [inline getting_started/setup/setup_up_2_board.rst]: @intelkevinputnam Can you help me understand the double '_ _' at the end of the link? It doesn't seem to fit with the [guidelines](https://thesofproject.github.io/latest/contribute/doc_guidelines.html#internal-cross-reference-linking). I left it alone b/c I wasn't sure what it was there for!

**Comment by stripes416** [comment]
Per @plb's suggestion a few months back, I tested BIOS v4.0 on my up2 board because the link in the public wiki for our suggested BIOS version pointed internally. I've not experienced any problems with it and @wenqingfu suggested we go ahead and change it to point externally.

File: getting_started/setup/setup_up_2_board.rst (hunk 1 above)
Comment (lgirdwood) [reply getting_started/setup/setup_up_2_board.rst]: @stripes416 the link works without the__ 
https://downloads.up-community.org/download/up-squared-uefi-bios-v4-0/

**Comment by intelkevinputnam** [comment]
LGTM. @stripes416 can you trim the extra underscore from the link before we merge?

**Comment by stripes416** [comment]
@intelkevinputnam Sorry -- Will do that now. Thanks for the reminder :)

**Pull request merged**